halo2_proofs/src/plonk/prover.rs

```diff
@@ -116,7 +118,7 @@ where
             if P::QUERY_INSTANCE {
                 let instance_commitments_projective: Vec<_> = instance_values
                     .iter()
-                    .map(|poly| params.commit_lagrange(poly, Blind::default()))
+                    .map(|poly| params.commit_lagrange(engine, poly, Blind::default()))
```

Comment:
We were thinking on removing the poly definitions/utilities from this crate and move them to an external one where we can have MV and UV as well as optimizations like parallelism and maybe ZAL then too. 

I'll follow-up on this. But would be nice to coordinate.